CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ _When adding new entries to the changelog, please include issue/PR numbers where
 * Simpler developer builds using CMake, see the [contributing notes](./CONTRIBUTING.md).
 * `kart log`: can now output the history of individual features using syntax `kart log -- <dataset-name>:feature:<feature-primary-key>`. [#496](https://github.com/koordinates/kart/issues/496)
 * Bugfix: fixed the error when merging a commit where every feature in a dataset is deleted. [#506](https://github.com/koordinates/kart/pull/506)
+* `kart index` now writes a simpler envelope-based index, rather than one that uses S2 cells. [#456](https://github.com/koordinates/kart/issues/456)
```

Comment:
did you mean `kart spatial-tree index` ?

I wonder if this is irrelevant detail in the changelog; the spatial index is a new feature since 0.10.x and isn't mentioned in the changelog yet at all (I think?), so mentioning the changed index type seems irrelevant. I'd just omit it, and probably add something about the new `spatial-tree` command in the 0.11 release.